src/cmd/run.rs

```diff
@@ -203,6 +203,19 @@ async fn poll_integration<D: Database>(
     db: &D,
     integration: &Integration,
     sender: &Queue<ScanGitVCSReference>,
+) -> Result<(), Error> {
+    loop {
+        if let Err(err) = execute_poll_integration(db, integration, sender).await {
+            warn!("attempt to poll integration failed: {err:#?}");
+        }
+    }
+}
+
+#[tracing::instrument(skip(db, sender))]
+async fn execute_poll_integration<D: Database>(
+    db: &D,
+    integration: &Integration,
+    sender: &Queue<ScanGitVCSReference>,
 ) -> Result<(), Error> {
```

Comment:
I know this isn't obvious, but `integration` is already traced via the caller (`poll_integration`). Please skip tracing it (again) here.

Reference for why this is important: https://fossa.atlassian.net/browse/ANE-1027

Basically, when you emit a trace span or event in a function, any context attached to parent spans is also included in that event; here's some example output: https://github.com/fossas/broker/pull/11

```suggestion
#[tracing::instrument(skip_all)]
async fn execute_poll_integration<D: Database>(
    db: &D,
    integration: &Integration,
    sender: &Queue<ScanGitVCSReference>,
) -> Result<(), Error> {
```